modules/client/examples/05-encoders-decoders/12-apply-installation.ts

```diff
@@ -46,32 +46,44 @@ const applyInstallationParams: ApplyInstallationParams = {
 
 const daoAddressOrEns: string = "0x123123123123123123123123123123123123"; // "my-dao.eth"
 
-const action: DaoAction = client.encoding.applyInstallationAction(
+const actions: DaoAction[] = client.encoding.applyInstallationAction(
   daoAddressOrEns,
   applyInstallationParams
 );
-console.log({ action });
+console.log({ actions });
 
 /* MARKDOWN
 Returns:
```

Comment:
Returns three actions:
- Grant root permission to the Plugin Setup Processor
- Ask it to apply the installation
- Revoke the root permission to the Plugin Setup Processor